apps/vaporgui/PCheckbox.h

```diff
@@ -1,12 +1,17 @@
 #pragma once
 
-#include "PWidget.h"
-#include <QCheckBox>
+#include "PLineItem.h"
 
-class PCheckbox : public PWidget {
+class VCheckBox;
+
+//! \class PCheckbox
+//! Creates a Qt Checkbox synced with the paramsdatabase using the PWidget interface.
+//! \copydoc PWidget
+
+class PCheckbox : public PLineItem {
     Q_OBJECT
     
-    QCheckBox *_qcheckbox;
+    VCheckBox *_qcheckbox;
```

Comment:
Unless there's a reason I don't see, could we rename _qcheckbox to _vcheckbox, since it's an instance of VCheckBox, not QCheckBox?

Reply:
done